homeassistant/components/tuya/light.py

```diff
@@ -526,6 +528,19 @@ def __init__(
         ):
             self._color_temp = int_type
             color_modes.add(ColorMode.COLOR_TEMP)
+            self._white_color_mode = ColorMode.COLOR_TEMP
+        elif (
+            workmode_type
+            := self.find_dpcode(  # If entity does not have color_temp, check if it has work_mode "white"
+                description.color_mode, dptype=DPType.ENUM, prefer_function=True
+            )
+        ):
```

Comment:
The formatting here is not great, I'd suggest to move the comment:
```suggestion
        # If entity does not have color_temp, check if it has work_mode "white"
        elif workmode_type := self.find_dpcode(
            description.color_mode, dptype=DPType.ENUM, prefer_function=True
        ):
```